### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: src/accounts/account.ts [hunk 1]
```diff
@@ -18,14 +18,6 @@ export class Account implements IAccount {
      */
     nonce: INonce = 0;
```
Comment (popenta) [inline src/accounts/account.ts:19]: this should also be changed to `BigInt`.

Comment (popenta) [reply src/accounts/account.ts:19]: And at some point, the class should be updated to specs.

File: src/networkProviders/apiNetworkProvider.ts [hunk 2]
```diff
@@ -146,17 +147,17 @@ export class ApiNetworkProvider implements INetworkProvider {
         return status;
     }
 
-    async sendTransaction(tx: ITransaction | ITransactionNext): Promise<string> {
+    async sendTransaction(tx: ITransaction): Promise<string> {
```
Comment (popenta) [inline src/networkProviders/apiNetworkProvider.ts:150]: I think `ITransactionNext` should have been deleted only when updating the `Transaction` class and use that here.

Comment (popenta) [reply src/networkProviders/apiNetworkProvider.ts:150]: I guess it's fine since you'll do it in the next PR.

**Review by popenta — APPROVED** [review]

File: src/accounts/account.ts (hunk 1 above)
Comment (danielailie) [reply src/accounts/account.ts:19]: if it's ok I'll do it in a different pr to keep this for remove only

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**